### PR TITLE
MONDRIAN-2335: SQL for dialect='postgres' in Measure expression is never used to generate native SQL

### PR DESCRIPTION
**Pull request by TatsianaKasiankova**
There is a fix proposed by Luc Boudreau and unit tests.

### CODE REVIEW

**Comment by TatsianaKasiankova** [comment]
@lucboudreau Hi, Luc
Could you please review this one.

File: testsrc/main/mondrian/rolap/sql/CodeSetTest.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
+/*
+// This software is subject to the terms of the Eclipse Public License v1.0
+// Agreement, available at the following URL:
+// http://www.eclipse.org/legal/epl-v10.html.
+// You must accept the terms of that agreement to use this software.
+//
+// Copyright (C) 2004-2005 Julian Hyde
+// Copyright (C) 2005-2015 Pentaho and others
```
Comment (lucboudreau) [inline testsrc/main/mondrian/rolap/sql/CodeSetTest.java]: Please change the copyright header to "2015-2015 Pentaho and others" only.

**Comment by lucboudreau** [comment]
Only 1 cosmetic change necessary. Will merge right after it's done.

**Comment by TatsianaKasiankova** [comment]
@lucboudreau 
Done

**Pull request merged**